tests/test_contact_form.py

```diff
@@ -1,15 +1,16 @@
-# from flask import url_for
-#
-#
-# def test_contact(user, client):
-#
-#     data = {
-#         'email': user['email'],
-#         'title': user['title'],
-#         'message': user['message']
-#     }
-#     resp = client.get(url_for('library.contact'))
-#     assert resp.status_code == 200
-#
-#     resp = client.post(url_for('library.contact'), data=data)
-#     assert resp.status_code == 200
+"""from flask import url_for
+
+
+def test_contact(user, client):
+
+    data = {
+        'email': user['email'],
+        'title': user['title'],
+        'message': user['message']
+    }
+    resp = client.get(url_for('library.contact'))
+    assert resp.status_code == 200
+
+    resp = client.post(url_for('library.contact'), data=data)
+    assert resp.status_code == 200
+"""
```

Comment:
In this example docstrings are unecessary :)
docstrings should be used in documentation stuff